clc/cmd/exec_context.go

```diff
@@ -314,13 +325,18 @@ func (s *simpleSpinner) Start() {
 	_ = s.sp.Start()
 }
 
+func (s *simpleSpinner) Stop() {
+	// ignoring the error here
+	_ = s.sp.Stop()
+}
+
 func (s *simpleSpinner) SetText(text string) {
 	s.text = text
 	if text == "" {
 		s.sp.Prefix("")
 		return
 	}
-	s.sp.Prefix(text + cancelMsg)
+	s.sp.Prefix("      " + text + cancelMsg)
```

Comment:
Shall we use strings.Repeat here to make it more readable?

Reply:
I've added that, but it made the code less clean.
I'll keep it if it's OK.